Automattic/pocket-casts-android
# Migrate device scoped settings to global settings

## Description

Some of the settings weren't synced globally. We decided that the client migration will be simpler than handling it on the server.

Internal ref: pdeCcb-4Kl-p2

## Testing Instructions

> [!note]
> This should be tested with staging. Use `debug` variant instead of `debugProd`.

Smoke test migrated settings. You don't have to do extensive testing. Check if the setting toggled is enough.

* Grid layout #1738 
* Auto archived starred #1700
* Volume boost #1746
* Row action #1748
* Files auto up next #1777
* Badges #1781
* Embedded artwork #1783
* Shelf actions #1787
* Delete cloud files #1864
* Delete local files #1863

## Checklist
 
#### I have tested any UI changes...
